Orange/widgets/classify/tests/test_owsvmclassification.py

```diff
@@ -0,0 +1,120 @@
+# Test methods with long descriptive names can omit docstrings
+# pylint: disable=missing-docstring
+import unittest
+from Orange.data import Table
+from Orange.widgets.classify.owsvmclassification import OWSVMClassification
+from Orange.widgets.tests.base import GuiTest
+
+
+class TestOWSVMClassification(GuiTest):
+    def setUp(self):
+        self.widget = OWSVMClassification()
+        self.event_data = None
+
+    # Check if right text is written for specific kernel
+    @unittest.skip("Not corrected widget on load")
```

Comment:
When we create widget and we do not click on any specific kernel all three (gamma, coeff and degree are visible) spinner boxes are visible until we do not select specific kernel.